tanner/utils/logger.py

```diff
@@ -4,16 +4,22 @@
 
 class Logger:
     @staticmethod
-    def create_logger(filename, logger_name):
-        log_filename = filename
+    def create_logger(debug_filename, err_filename, logger_name):
         logger = logging.getLogger(logger_name)
         logger.setLevel(logging.DEBUG)
-        handler = logging.handlers.RotatingFileHandler(log_filename, encoding='utf-8')
-
         formatter = logging.Formatter(fmt='%(asctime)s %(levelname)s:%(name)s:%(funcName)s: %(message)s',
                                       datefmt='%Y-%m-%d %H:%M')
-        handler.setFormatter(formatter)
 
-        logger.addHandler(handler)
+        # DEBUG log to 'tanner.log'
+        debug_log_handler = logging.handlers.RotatingFileHandler(debug_filename, encoding='utf-8')
+        debug_log_handler.setFormatter(formatter)
```

Comment:
Maybe we need to set level for debug_log_handler? Errors are still in the tanner.log :)
```
afeena@ubuntu:/opt/tanner$ tail -n 2 tanner.log
2017-03-23 10:58 ERROR:tanner.rfi_emulator.RfiEmulator:get_rfi_result: Error during connection to php sandbox [Errno 111] Cannot connect to host 127.0.0.1:8088 ssl:False [Can not connect to 127.0.0.1:8088 [Connection refused]]
2017-03-23 10:58 INFO:tanner.server.HttpRequestHandler:handle_event: TANNER response b'{"version": 1, "response": {"message": {"sess_uuid": "6fb6697d-101f-44df-8cb9-605d7ebdde91", "detection": {"payload": "", "order": 2, "name": "rfi"}}}}'

afeena@ubuntu:/opt/tanner$ tail -n 1 tanner.err
2017-03-23 10:58 ERROR:tanner.rfi_emulator.RfiEmulator:get_rfi_result: Error during connection to php sandbox [Errno 111] Cannot connect to host 127.0.0.1:8088 ssl:False [Can not connect to 127.0.0.1:8088 [Connection refused]]
```